Baidak_Max/HW5/Task_2.py

```diff
@@ -0,0 +1,16 @@
+number = int(input("Enter a number:\n"))
```

Comment:
good job
look at this implementation:

number = int(input("Enter a number:\n"))
if number < 0:
    print("Enter a positive number:")
else:
    a, b = 0, 1
    fibonacci = [a, b]
    while b <= number:
        a, b = b, a + b
        fibonacci.append(b)

    print(f"Fibonacci numbers up to {number}:")
    print(*fibonacci)